WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/FeatureConfig/EndToEndIdentityFeatureConfigDecoder.swift

```diff
@@ -31,7 +31,9 @@ struct EndToEndIdentityFeatureConfigDecoder {
         return EndToEndIdentityFeatureConfig(
             status: payload.status,
             acmeDiscoveryURL: payload.config.acmeDiscoveryURL,
-            verificationExpiration: payload.config.verificationExpiration
+            verificationExpiration: payload.config.verificationExpiration,
+            crlProxy: nil,
+            useProxyOnMobile: nil
```

Comment:
I think we may need to parse these too.

Reply:
Same comment as above, these were introduced in v6, are we sure we always receive values for it ?

Reply:
as per backend team feedback, these are not part of the response, so we just assign nil to crlProxy and default to false for useProxyOnMobile